vagrant/Makefile

```diff
@@ -44,10 +57,11 @@ helm-dependencies:
 	helm dependency update "${helm_path}"
 
 helm-upgrade:
+	$(eval local_values_param := $(shell [ -f ${local_values_file} ] && echo "-f ${local_values_file}" || echo ""))
```

Comment:
`eval` and `shell` 🙀

Reply:
This seems to be the way to define variables evaluated at rule execution time: https://stackoverflow.com/questions/1909188/define-make-variable-at-rule-execution-time. The Makefile's eval function: https://www.gnu.org/software/make/manual/make.html#Eval-Function